src/main/java/com/st/utopia/cancellationservice/dao/TicketDao.java

```diff
@@ -0,0 +1,49 @@
+package com.st.utopia.cancellationservice.dao;
+
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+//import org.springframework.data.jpa.repository.Modifying;
+//import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import com.st.utopia.cancellationservice.model.Ticket;
+import com.st.utopia.cancellationservice.model.TicketIdentity;
+
+@Repository
+public interface TicketDao extends JpaRepository<Ticket, TicketIdentity>{
+	/**
+	 * Get the ticket with a specific flight and row and seat in the plane .
+	 *
+	 */
+	default Ticket getTicket(Integer flight, Integer row, Character seat) {
+		final Optional<Ticket> record = findById(new TicketIdentity(flight, row, seat));
+		if (record.isPresent()) {
+			return record.get();
+		} else {
+			return null;
+		}
+	}
+	
+// @Modifying(clearAutomatically = true)

```

Comment:
In general, remove code rather than commenting it out when committing; if you're using version control properly, it is very easy to recover an old version from the history if it turned out you wanted it back.

Reply:
I agree, and it makes your code easier to read, for me at least